sentinel2-processing/prepare_10m_sentinel_imagery.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+"""
+This is a Python version of a bash script written by Rebecca Clarke https://gist.github.com/rebclarke/b9e55a3558be3ec76248c78379b67366
+"""
+
+import os
+import glob
+import shutil
+from osgeo import gdal
```

Comment:
It doesn't look like you're using this import anywhere. Did you mean to change the `subprocess` uses below to use this library? That would be preferable.